docs/example_usage/breeds_datasets.rst

```diff
@@ -0,0 +1,288 @@
+Creating BREEDS subpopulation shift benchmarks
+===============================================
+
+In this document, we will discuss how to create BREEDS datasets, used in 
+the `code <https://github.com/MadryLab/BREEDS_benchmarks>`_ `release
+<https://github.com/MadryLab/BREEDS_benchmarks>`_ based on our paper [STM20]_. 
+These datasets can be used to study model robustness to subpopulation 
+shift---a specific form of distribution shift wherein the subpopulations of data 
+(e.g., animal breeds) present in the training set (source domain) are 
+entirely different from the test set (target domain). 
+You can create BREEDS datasets from any existing dataset that has 
+an explicit class hierarchy describing the semantic relationship 
+between pairs of classes (e.g., ImageNet, OpenImages). 
+In this walkthrough, we will do so using ImageNet and the corresponding
+class hierarchy from [STM20]_ (modification of WordNet to be better-suited
+for the object recognition task).
+
+At a high-level, the procedure to construct BREEDS datasets is as follows:
+
+1. Use the dataset's class hierarchy to define `superclasses` by grouping
+   together semantically similar dataset classes (e.g., define the `dog`` superclass 
+   by aggregating all the dog breed classes). 
+2. Design the classification task to be between different superclasses, so that
+   we have easy access to the individual subpopulations (i.e., the original 
+   dataset classes). Then, for each superclass, we will partition the subpopulations 
+   into the source (used for model training) and target domain without overlap
+   (used for evaluation). 
```

Comment:
Suggested edit:

1. Use the ImageNet hierarchy to group together similar classes into "superclasses," e.g., we can make a "dog" superclass by ...
2. Partition the subclasses of each superclass into "training breeds" and "testing breeds." The classification task will be differentiating between the superclasses, but at train time we will see only the "training breeds" from each superclass, and at testing time we will see only the testing breeds.